examples/streamlit/README.md

```diff
@@ -2,8 +2,12 @@
 
 This is an example on how to deploy a streamlit app using `pynsist`. The example also includes the required packages to run plotly inside the app.
 
+The extra wheels of the packages `blinker` and `pyarrow` are required (can't be found on Pypi).
```

Comment:
Pyarrow seems to have wheels on PyPI:

https://pypi.org/project/pyarrow/5.0.0/#files

`pyarrow-5.0.0-cp38-cp38-win_amd64.whl` is the one it should find.

Reply:
I missed it, tested by building the app without the `pyarrow` in the `extra_wheels` folder and it works. Removed from the readme.